wifiphisher/common/interfaces.py

```diff
@@ -465,6 +506,18 @@ def set_interface_mode(self, interface_name, mode):
         pyw.modeset(card, mode)
         pyw.up(card)
 
+    def add_card(self, interface):
```

Comment:
it would be like so:
``` python
def is_interface_valid(self, interface_name, mode=None)
    ...
    self._active.add(interface_name)
```

And maybe we should do a check to see if interface is connected to internet?